### PR TITLE
Update in-korean.md

### PR DESCRIPTION
**Pull request by lukeyang**
Some editorial changes proposal and fixed broken link for PDF files.

### Description
Editorial changes for Korean sentences and fixing broken link for PDF whitepaper files.

### Test Plan

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/3520)

### CODE REVIEW

**Review by davidiw — COMMENTED** [review]
The links work for me on the original but not this one. What happens when you click the links? @bchocho / @junkil-park can you validate the Korean contributions?

**Comment by bchocho** [comment]
@davidiw really? It's the opposite for me. Only the updated links work.
* Original: https://github.com/aptos-labs/aptos-core/blob/main/papers/whitepaper-korean.pdf (I get a 404)
* Updated: https://github.com/aptos-labs/aptos-core/blob/main/developer-docs-site/static/papers/whitepaper-korean.pdf

I'll take a look at the Korean changes :)

**Comment by davidiw** [comment]
oh jeez... these links are for the website not for github ... this is one of the disadvantages of docusaurus @rajkaramchedu -- something to look into.

**Review by bchocho — CHANGES_REQUESTED** [review]
수정 감사합니다! 코멘트 달았습니다.

File: developer-docs-site/docs/aptos-white-paper/in-korean.md [hunk 1]
```diff
@@ -45,6 +45,6 @@ slug: "aptos-white-paper-in-korean"
 
 :::tip 전체 PDF 버전
 
-- **초록**: Aptos 백서의 한국어 버전 전체 PDF를 보려면 [여기를 클릭하십시오](/papers/whitepaper-korean.pdf).
-- **English**: Click [here for the full PDF of the Aptos White Paper](/papers/Aptos-Whitepaper.pdf).
+- **초록**: Aptos 백서의 한국어 버전 전체 PDF를 보려면 [여기를 클릭하십시오](/developer-docs-site/static/papers/whitepaper-korean.pdf).
```
[the remark this answers is not in the document]
Comment (rajkaramchedu) [reply developer-docs-site/docs/aptos-white-paper/in-korean.md:48]: Why is this linked to HTML page when the admonition clearly says these are links to the PDF docs? This correction is incorrect. Docusaurus links should be done in a specific way, just see how the original is "/papers/...".

File: developer-docs-site/docs/aptos-white-paper/in-korean.md [hunk 2]
```diff
@@ -45,6 +45,6 @@ slug: "aptos-white-paper-in-korean"
...
+- **English**: Click [here for the full PDF of the Aptos White Paper](/developer-docs-site/static/papers/Aptos-Whitepaper.pdf).
```
Comment (rajkaramchedu) [inline developer-docs-site/docs/aptos-white-paper/in-korean.md:49]: Incorrect link. Original link is correct and works.

**Review by rajkaramchedu — CHANGES_REQUESTED** [review]

**Comment by rajkaramchedu** [comment]
@lukeyang you said, "fixed broken link for PDF files." Which browser are you using? What exactly happens when you click on the PDF link? These links are correct and they were tested working before publishing. Please be specific in how they are broken for you so I can look into it. kamsamida :- )

**Comment by lukeyang** [comment]
@rajkaramchedu 

After the release of Korean white paper, pdf release was widely spread in the Korean community. But the spread link was https://github.com/aptos-labs/aptos-core/blob/main/developer-docs-site/static/papers/whitepaper-korean.pdf, not https://aptos.dev/assets/files/whitepaper-korean-67e84101467803b96c4f481a1161ec45.pdf

As a native Korean, I wanted to correct some Korean sentences. During reviewing the documents, I thought that the pdf link was broken, because of my misunderstanding of docusaurus.

What I thought the link is broken in this document release : https://github.com/aptos-labs/aptos-core/blob/main/papers/whitepaper-korean.pdf 
What I thought the link should be fixed : https://github.com/aptos-labs/aptos-core/blob/main/developer-docs-site/static/papers/whitepaper-korean.pdf

But I saw the comments from @bchocho the the link is correct in https://aptos.dev link. My correct suggestion for the link was wrong. Sorry for confusion. Only some minor Korean sentences correction make sense.

FYI, I use Chome in Mac m1 machine.

**Comment by rajkaramchedu** [comment]
Perfect, thanks, @lukeyang for the contextual explanation. I see that now. Okay, so if you now push the changes only to the Korean sentences and retain the original PDF links, I will approve this and we can land this asap. Thanks again!

**Review by bchocho — CHANGES_REQUESTED** [review]
Can you please also change the last instance of 어플리케이션 on line 36?

**Review by bchocho — APPROVED** [review]
Thank you!

**Comment by bchocho** [comment]
@rajkaramchedu when you get the chance can you accept as well? :)

**Review by rajkaramchedu — APPROVED** [review]

**Pull request merged**